internal/provider/notification_recipients.go

```diff
@@ -18,9 +18,9 @@ import (
 	"github.com/honeycombio/terraform-provider-honeycombio/internal/models"
 )
 
-func notificationRecipientSchema() schema.SetNestedBlock {
+func notificationRecipientSchema(allowedTypes []client.RecipientType) schema.SetNestedBlock {
```

Comment:
This fixes the bug introduced by the refactor

Reply:
Good catch!!